css/responsive.css

```diff
@@ -0,0 +1,125 @@
+/* Beginning of Layout Framework styles */
+.row {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.column {
+    width: 100%;
+    margin-bottom: 10px;
```

Comment:
why not remove this and apply margin-bottom to specific section or elements, instead of having this general margin on all columns.